new-lamassu-admin/src/components/inputs/base/TextInput.js

```diff
@@ -60,7 +110,7 @@ const TextInput = memo(
           ) : null
         }}
         InputLabelProps={{ className: classes.labelRoot }}
-        {...props}
+        {...R.omit(['className'], props)}
```

Comment:
Why is className getting ommited and not used on the main parent? As it stands this prop does nothing.

Reply:
`props.className` is omitted because it is already passed into the classNames object on line 86, which is used on line 100. If it was not omitted, the child component would receive **two** `className` props and there would be a conflict. This is visible on the Phone Number input where, if `props.className` is not omitted, the label has the wrong color when the input is empty.

Reply:
Ahh, just saw that. Instead of using props.className you can desconstruct that.

`  ({ name, onChange, onBlur, value, error, suffix, large, ...props }) => {`
`  ({ name, onChange, onBlur, value, error, suffix, large, className, ...props }) => {`

and then you don't need to omit it.

Reply:
Ah, good point. Changed in https://github.com/lamassu/lamassu-server/pull/354/commits/cb69441239bf2b56da02775aad37a5a16c28fe85